pkg/acsengine/defaults-kubelet.go

```diff
@@ -97,6 +96,22 @@ func setKubeletConfig(cs *api.ContainerService) {
 		setMissingKubeletValues(cs.Properties.MasterProfile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
 		addDefaultFeatureGates(cs.Properties.MasterProfile.KubernetesConfig.KubeletConfig, o.OrchestratorVersion, "", "")
 
+		var defaultMasterMaxPods int
+		// set defaultMasterMaxPods to OrchestratorProfile.KubernetesConfig.MaxPods if exist.
+		if o.KubernetesConfig != nil && o.KubernetesConfig.MaxPods != 0 {
+			defaultMasterMaxPods = o.KubernetesConfig.MaxPods
+		} else {
+			if o.IsAzureCNI() {
+				defaultMasterMaxPods = DefaultKubernetesMaxPodsVNETIntegrated
+			} else {
+				defaultMasterMaxPods = DefaultKubernetesMaxPods
```

Comment:
Is this supposed to be `DefaultKubernetesKubeletMaxPods` instead of `DefaultKubernetesMaxPods`?

Reply:
I think this is due to the historical changes discusses in the previous comments. Right now, we are converging to use DefaultKubernetesMaxPodsVNETIntegrated for Azure CNI and DefaultKubernetesMaxPods for Kubenet. So DefaultKubernetesKubeletMaxPods can be retired.

Reply:
Sounds good